src/Chapter1.hs

```diff
@@ -520,7 +521,14 @@ branches because it is an expression and it must always return some value.
   satisfying the check will be returned and, therefore, evaluated.
 -}
 closestToZero :: Int -> Int -> Int
-closestToZero x y = error "closestToZero: not implemented!"
+closestToZero x y = min (abs x) (abs y)
```

Comment:
This will return the absolute value of the number instead of the number itself. But you got the right idea!